package.json

```diff
@@ -11,11 +11,13 @@
     "test": "npm run test:unit",
     "test:unit": "jest ./src/*",
     "test:e2e": "jest ./end-to-end-tests/* --runInBand",
-    "prepublish": "npm run build:cdn"
+    "prepublishchecks": "npm run lint && npm run test:unit && npm run build:cdn && npm run build:esm && npm run test:e2e",
+    "prepublish": "publish-please guard",
```

Comment:
I believe `prepublish` is deprecated, and you should use either `prepare` or `prepublishOnly`